### PR TITLE
3DS2 documentation ready

### PR DESCRIPTION
**Pull request by manuel-huez**

### CODE REVIEW

File: source/migrating-to-processout.md.erb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+title: Migrating to ProcessOut
+---
+
+# Migrating to ProcessOut
+
+Most of the time, if you operate a SaaS or recurring business, or even an eCommerce
+that provides one-click checkouts, you have tokenized your customers cards in
+your PSPs Vault.
+
+Because ProcessOut is a PCI-DSS Service Provider Level 1 ([see our certification here &nearr;](https://www.processout.com/documents/pci-dss-aoc.pdf)), we can store card details for you in the PSP Agnostic ProcessOut Vault. We can therefore make it easy for
```
[the remark this answers is not in the document]
Comment (manuel-huez) [reply source/migrating-to-processout.md.erb]: Is there any security concerns for that? I assumed it was OK as the AOC doesn't contain any information regarding our infrastructure, and it reassures merchants looking at the migration documentation :)

File: source/migrating-to-processout.md.erb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+your PSPs Vault.
+
+Because ProcessOut is a PCI-DSS Service Provider Level 1 ([see our certification here &nearr;](https://www.processout.com/documents/pci-dss-aoc.pdf)), we can store card details for you in the PSP Agnostic ProcessOut Vault. We can therefore make it easy for 
+you to switch over to ProcessOut by syncing with your old Payment Service Provider 
+and importing the card numbers they stored on their side. This allows you to 
+provide uninterrupted service to your customers as you switch over your payments 
+to ProcessOut.
+
+Please note that ProcessOut will only import Card objects from your old PSPs. 
+Re-creating Customer, Plans or Subscriptions objects should be done at your own discretion.
```
[the remark this answers is not in the document]
Comment (manuel-huez) [reply source/migrating-to-processout.md.erb]: I feel like the old version is a bit friendlier?

File: source/payments/payments-and-sca.md.erb [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+```
+</div>
+
+If the capture was successful, a transaction is returned. You may then check that
+its `status` attribute is set to `completed` in order to confirm the payment.
+Learn more about the transaction's statuses [here &nearr;](/payments/handle-status-changes-webhooks/#transaction-statuses).
+
+## Step 6: Listen to webhooks for payment updates
+
+Although ProcessOut.js supports a synchronous flow, allowing you to offer a greater
```
[the remark this answers is not in the document]
Comment (manuel-huez) [reply source/payments/payments-and-sca.md.erb]: This doesn't seem to fit the end of the sentence

File: source/migrating-to-processout.md.erb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+step to ensure that when we initiate the import from your old PSP, we're sure
+that no new cards will be created in your old PSP, which would de-synchronize
+the import and miss cards leading to lost payments.
+
+## Step 2: Intro your ProcessOut Account Manager to your old PSP
+
+Now that you've launched ProcessOut in your application so that all new customers and
+cards get sent and processed on ProcessOut, you can intro your ProcessOut Account
+Manager to your old PSP Account Manager. We'll synchronize with them to initiate
+the cards export. This is a manual step that can take from a few days to 2 weeks
```
[the remark this answers is not in the document]
Comment (manuel-huez) [reply source/migrating-to-processout.md.erb]: Shouldn't this be in the plural form?

[the remark this answers is not in the document]
Comment (louis-paul) [reply source/migrating-to-processout.md.erb]: I don’t know the exact rule but we say “the car park,” not “the cars park” :-)

File: source/migrating-to-processout.md.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (louis-paul) [reply source/migrating-to-processout.md.erb]: It is not meant to be public.

File: source/payments/payments-and-sca.md.erb (hunk 3 above)
[the remark this answers is not in the document]
Comment (louis-paul) [reply source/payments/payments-and-sca.md.erb]: > Although ProcessOut.js supports a synchronous flow, allowing you to give your users a better experience, we strongly recommend that you also set up an endpoint to receive the webhooks we send to be notified of payment updates.

That seems okay to me with the rest of the changes? Maybe we can split the sentence to make it shorter?

File: source/migrating-to-processout.md.erb (hunk 4 above)
[the remark this answers is not in the document]
Comment (manuel-huez) [reply source/migrating-to-processout.md.erb]: Yes but in this case "park" is used as a verb, whereas here we're using "export" as a noun. And in the export we have several cards

File: source/payments/payments-and-sca.md.erb [hunk 5]
```diff
@@ -0,0 +1,510 @@
...
+meant to define how much your customer will get charged, and set metadata
+related to the payment. This Invoice will also be sent back to your front-end
+to properly authenticate your customer if required by local regulations.
+
+<div class="language-selector-wrapper">
+<%= partial "lg-selector" %>
+```shell
+curl -X POST https://api.processout.com/invoices \
+    <%= partial "header-curl" %> \
+    -d name="Awesome invoice" \
```
[the remark this answers is not in the document]
Comment (manuel-huez) [reply source/payments/payments-and-sca.md.erb]: Awesome invoice is what we're using everywhere on the API docs so I'd rather keep it like that for now

File: source/migrating-to-processout.md.erb (hunk 4 above)
[the remark this answers is not in the document]
Comment (manuel-huez) [reply source/migrating-to-processout.md.erb]: https://www.englishclub.com/grammar/nouns-compound.htm

"tool box" is a good example to show there shouldn't be a s there- you're right

File: source/payments/payments-and-sca.md.erb [hunk 6]
```diff
@@ -0,0 +1,510 @@
...
+    -d amount="9.99" \
+    -d currency=USD 
+```
+
+```javascript
+<%= partial "header-js" %>
+client.newInvoice().create({
+    name:     "Amazing item",
```
Comment (louis-paul) [inline source/payments/payments-and-sca.md.erb]:
```suggestion
    name:     "Awesome invoice",
```

File: source/payments/payments-and-sca.md.erb [hunk 7]
```diff
@@ -0,0 +1,510 @@
...
+}, function(err) {
+    // An error occured
+
+});
+```
+
+```python
+<%= partial "header-python" %>
+invoice = client.new_invoice().create({
+    "name":     "Amazing item",
```
Comment (louis-paul) [inline source/payments/payments-and-sca.md.erb]:
```suggestion
    "name":     "Awesome invoice",
```

File: source/payments/payments-and-sca.md.erb [hunk 8]
```diff
@@ -0,0 +1,510 @@
...
+    "amount":   "4.99",
+    "currency": "USD"
+})
+```
+
+```ruby
+<%= partial "header-ruby" %>
+invoice = client.invoice.create(
+    name:     "Amazing item",
```
Comment (louis-paul) [inline source/payments/payments-and-sca.md.erb]:
```suggestion
    name:     "Awesome invoice",
```

File: source/payments/payments-and-sca.md.erb [hunk 9]
```diff
@@ -0,0 +1,510 @@
...
+    amount:   "4.99",
+    currency: "USD"
+)
+```
+
+```php
+<%= partial "header-php" %>
+$invoice = $client->newInvoice()->create(array(
+    "name"     => "Amazing item",
```
Comment (louis-paul) [inline source/payments/payments-and-sca.md.erb]:
```suggestion
    "name"     => "Awesome invoice",
```

File: source/payments/payments-and-sca.md.erb [hunk 10]
```diff
@@ -0,0 +1,510 @@
...
+    "amount"   => "4.99",
+    "currency" => "USD"
+));
+```
+
+```go
+<%= partial "header-go" %>
+iv, err := client.NewInvoice().Create(processout.InvoiceCreateParameters{
+    Invoice: &processout.Invoice{
+        Name:     processout.String("Amazing item"),
```
Comment (louis-paul) [inline source/payments/payments-and-sca.md.erb]:
```suggestion
        Name:     processout.String("Awesome invoice"),
```

File: source/payments/payments-and-sca.md.erb (hunk 5 above)
[the remark this answers is not in the document]
Comment (louis-paul) [reply source/payments/payments-and-sca.md.erb]: So let’s make the other commands match instead? I’ve made suggestions for those.

File: source/migrating-to-processout.md.erb [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+<hr>
+
+## Step 1: Preparing your ProcessOut integration
+
+Before getting your cards exported from your old PSP to ProcessOut, you’ll need
+to integrate ProcessOut into your application. This can be done by following
+our integration guides and testing environment.
+
+Learn more about how to tokenize new cards with [ProcessOut.js &nearr;](/payments/processoutjs/), [create Customer
+Tokens from them](/payments/save-token-and-capture-later/) and use our [Subscriptions engine &nearr;](/subscriptions/get-started/).
```
Comment (louis-paul) [inline source/migrating-to-processout.md.erb]:
```suggestion
Tokens from them](/payments/save-token-and-capture-later/) and use our [Subscription engine &nearr;](/subscriptions/get-started/).
```

File: source/payments/payments-and-sca.md.erb [hunk 12]
```diff
@@ -0,0 +1,509 @@
...
+      // Enable back the button
+      document.getElementById("paymentBtn").disabled = false;
+    });
+
+    return false;
+  });
+}
+```
+
+We bound the form but we also want to implement the `processoutCardTokenized`
```
Comment (louis-paul) [inline source/payments/payments-and-sca.md.erb]:
```suggestion
We have bound the form but we also want to implement the `processoutCardTokenized`
```

**Pull request merged**